### ISSUE TITLE
Global service ID conflicts when m2if is used inside another Symfony app

### ISSUE THREAD

**Issue by adhocore** [techdivision/import#166]
Some service IDs like `configuration`, `loggers`, `connection` are used in global context:
ref: https://github.com/techdivision/import/blob/15.x/symfony/Resources/config/services.xml

it would be best to have them namespaced like so:
`import.configuration`, `import.loggers`, `import.connection`

to not break BC, we can alias them to their original IDs.

(PS: there can be more such service IDs than mentioned above)

**Comment by wagnert** [comment]
@adhocore You're right, this are technical debts we've to solve. I'll add this to the next release.
